pkgs/development/libraries/pipewire/default.nix

```diff
@@ -69,7 +64,7 @@ let
 
   self = stdenv.mkDerivation rec {
     pname = "pipewire";
-    version = "0.3.45";
+    version = "0.3.43";
```

Comment:
We can merge this after you reverted the accidental pipewire downgrade.

Reply:
Oops, my bad.

Reply:
@SuperSandro2000 fixed, 0.3.45 seems to spit an error saying that it cannot find `x11-xcb`, though.

Reply:
Weirdly enough, adding the SDL2 back to the buildInputs seem to fix the problem, even with "-Dsdl2=disabled".